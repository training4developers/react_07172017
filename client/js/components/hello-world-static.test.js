import React from 'react';
import renderer from 'react-test-renderer';
import { render } from 'enzyme';

import { HelloWorld } from './hello-world';

describe('<HelloWorld /> React Test Renderer Static HTML', () => {

  it('<HelloWorld /> renders', () => {

    const tree = renderer.create(<HelloWorld />).toJSON();
    expect(tree).toMatchSnapshot();

  });

});

describe('<HelloWorld /> Enzyme Static HTML', () => {

  it('<HelloWorld /> renders', () => {
    const component = JSON.stringify(render(<HelloWorld />).html());
    expect(component).toMatchSnapshot();
  });

});
