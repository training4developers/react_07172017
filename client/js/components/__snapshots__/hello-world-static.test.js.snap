// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloWorld /> Enzyme Static HTML <HelloWorld /> renders 1`] = `"\\"<h1>Hello World!</h1>\\""`;

exports[`<HelloWorld /> React Test Renderer Static HTML <HelloWorld /> renders 1`] = `
<h1>
  Hello World!
</h1>
`;
